### PR TITLE
[feat-5540]: Update split form styling

### PR DESCRIPTION
**Pull request by alimpens**
Ticket: [SIG-5540](https://gemeente-amsterdam.atlassian.net/browse/SIG-5540)

To discuss:

- The styling file for the incident split components is outside of the components folder. What's the convention here?
- The 'Extra deelmelding toevoegen'-button is wrapped in a `fieldset`, that seems a little odd. It's not really a form section, right?
- Is there a reason not to use the ASC `breakpoint` utility? It's not used everywhere.

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

[SIG-5540]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5540?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentSplitContainer/styled.js [hunk 1]
```diff
@@ -73,8 +73,34 @@ export const StyledLabel = styled(Label)`
...
+export const StyledButtonContainer = styled.div`
+  grid-column: span 7;
+`
+
 export const StyledFieldset = styled.fieldset`
+  background-color: ${themeColor('tint', 'level3')};
   scroll-margin-top: ${themeSpacing(15)};
+  grid-column: span 12;
+  padding: ${themeSpacing(5)};
+  margin-inline: -${themeSpacing(5)};
```
Comment (SireeKoolenWijkstra) [inline src/signals/incident-management/containers/IncidentSplitContainer/styled.js:99]: This is a negative margin, which isn't used by SIgnalen.

File: src/signals/incident-management/containers/IncidentSplitContainer/styled.js [hunk 2]
```diff
@@ -73,8 +73,34 @@ export const StyledLabel = styled(Label)`
...
+  display: grid;
+  column-gap: ${themeSpacing(16)};
+  row-gap: ${themeSpacing(8)};
+  grid-template-columns: 7fr 5fr;
 `
 
 export const StyledHeading = styled(Heading)`
```
Comment (SireeKoolenWijkstra) [inline src/signals/incident-management/containers/IncidentSplitContainer/styled.js:106]: Lineheight is 30px but should b 24 px.

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]
- [ ] 
image These two boxes should line out on the right side.

File: src/signals/incident-management/containers/IncidentSplitContainer/styled.js (hunk 2 above)
Comment (alimpens) [reply src/signals/incident-management/containers/IncidentSplitContainer/styled.js:106]: Hm, I see that's in the design, but would mean a break from the Amsterdam brand style. I'll ask Linda if this is intentional

File: src/signals/incident-management/containers/IncidentSplitContainer/styled.js (hunk 1 above)
Comment (alimpens) [reply src/signals/incident-management/containers/IncidentSplitContainer/styled.js:99]: Do we use another pattern to allow for background color overflow?

File: src/signals/incident-management/containers/IncidentSplitContainer/styled.js (hunk 2 above)
Comment (alimpens) [reply src/signals/incident-management/containers/IncidentSplitContainer/styled.js:102]: Checked with Linda, it was unintentional

**Review by vdegraaf — APPROVED** [review]

**Comment by vdegraaf** [comment]
@alimpens one question: what was the end result of the missing label of the textarea? I see it's still not there.

**Comment by alimpens** [comment]
> @alimpens one question: what was the end result of the missing label of the textarea? I see it's still not there.

i checked with Oscar, he couldn't really remember why the label was removed. Shall I make a new ticket for that? Or fix it in this PR?

**Comment by alimpens** [comment]
To summarize the discussion in person:

- I'll fix the folder structure in a seperate PR
- I'll create a new ticket to fix the missing labels

File: src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/IncidentSplitForm.tsx [hunk 3]
```diff
@@ -67,7 +68,7 @@ const IncidentSplitForm: FC<IncidentSplitFormProps> = ({
   onSubmit,
   isSubmitting,
 }) => {
-  const maxNoteLength = 1000
+  const maxNoteLength = 3000
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/IncidentSplitForm.tsx:71]: This is still 1000 right?

Comment (alimpens) [reply src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/IncidentSplitForm.tsx:71]: [The design](https://www.figma.com/file/HCZLFhdGNf6s6NnQeZYuKZ/Schetsen?type=design&node-id=904-1080&mode=design&t=pEjc6a3yGcU7OHfH-0) specifies 3000, I figured that was what the acc criterion 'aantal tekens in text area' meant?

**Pull request merged**